addons/transformations.md

```diff
@@ -60,29 +59,29 @@ Transformation files need to be placed in the directory `$OPENHAB_CONF/transform
 
 To keep these examples simple, the contents of the referenced files `window_esp.map` and `convert-C-to-F.js` were left out.
 
-## `SCRIPT` Transformation
+## Script Transformation
 
-The `SCRIPT` transformation is available from the framework and needs no additional installation.
+The script transformation is available from the framework and needs no additional installation.
 It allows transforming values using any of the available scripting languages in openHAB (JSR-223 or DSL).
 
-The script needs to be placed in the `$OPENHAB_CONF/transform` folder with an extension `.script` regardless of the actual script type.
-When referencing a transformation, the script type must be prepended to the filename (e.g. `dsl:stringlength.script` for the DSL version of `stringlength.script`).
-Please note that you cannot have transformations with the same name and different languages as the file-extension is always `script`.
-The script type depends on the scripting engine used, it is usually either the file extension or the MIME-type of the scripts.
+The script needs to be placed in the `$OPENHAB_CONF/transform` folder with the native extension for the chosen language, for example `stringlength.js` for a
+transformation using JS Scripting. The script file name here acts as the `script identifier` for the
```

Comment:
Pls, don't use line breaks in between. Pls, put each sentence in its own line like

... for a transformation using JS Scripting.
The script file name ...